src/main/frontend/pipeline-console-view/pipeline-console/main/pipeline-console.scss

```diff
@@ -88,6 +88,19 @@ g.build-status-icon__outer {
   transform: translate(0, 0) !important;
 }
 
+.icon-blue-anime .build-status-icon__outer,
+.icon-red-anime .build-status-icon__outer,
+.icon-yellow-anime .build-status-icon__outer,
+.icon-aborted-anime .build-status-icon__outer,
+.icon-disabled-anime .build-status-icon__outer,
+.icon-grey-anime .build-status-icon__outer,
+.icon-nobuilt-anime .build-status-icon__outer {
+  .svg-icon {
+    // Ensures spinning icons spin around the center.
+    transform-origin: center !important;
```

Comment:
not sure if these css changes or another one are 100% working, I saw an inprogress graph circle rotate off the dots while running, can be reproduced with https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/220#issuecomment-1751878453

<img width="231" alt="image" src="https://github.com/jenkinsci/pipeline-graph-view-plugin/assets/21194782/96e9e5a3-b34a-44d5-9095-2cd7ecdc1eb0">

Reply:
All the CSS changes should be dropped from this PR (it's either an old change I was playing with or a bad merge).